workbench-core/accounts/src/schemas/awsAccountTemplateUrls.ts

```diff
@@ -0,0 +1,19 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+// Schema for GetTemplate API
+import { Schema } from 'jsonschema';
+
+const AwsAccountTemplateUrlsSchema: Schema = {
+    id: '/getTemplate',
```

Comment:
technically we name schemas after API - so that should be 'awsAccountTemplateUrls'. Also @nguyen102 started pushing for using Zod for model validation. I'm fine with doing jsonschema now and add a task to switch to Zod in follow up PR. @nguyen102 any strong opinion here?

Reply:
@pbabz if you have time, Zod validation would be preferred. But if not we can do it in a follow up PR.